src/main/scala/vsys/contract/DataType.scala

```diff
@@ -0,0 +1,30 @@
+package vsys.contract
+
+object DataType extends Enumeration {
+  val PublicKey = Value(1)
+  val Address = Value(2)
+  val Amount = Value(3)
+  val Int32 = Value(4)
+  val ShortText = Value(5)
+  val ContractAccount = Value(6)
+  val Account = Value(7)
+
+  def fromByte(b: Byte): Option[DataType.Value] = {
+    if (b < DataType.PublicKey.id || b > DataType.Account.id)
+      None
+    else
+      Some(DataType(b))
+  }
+
+  private def check(a: Byte, b: Byte): Boolean = {
+    if (a == b) true
+    else if (a == DataType.Account.id) b == DataType.Address.id || b == DataType.ContractAccount.id
```

Comment:
if a == DataType.Address.id and b == DataType.ContractAccount.id, what should be the return value?

Reply:
false.
`Account` is a data type include `Address` and `ContractAccount`
this will be used as input parameter type in functions and operation codes
but this reminder me that we should take care the check here.

@Icermli 

if opc/function require `Account` type, we can accept both `Address` and `ContractAccount`, but if opc/function require `Address` or `ContractAccount` type, we can not accept `Account`.

however, we will not accept a detail `DataEntry` which type is `Account`, since `Account` type **only** used as input parameter type in opcs and functions

Reply:
ok